lib/ansible/modules/cloud/amazon/rds.py

```diff
@@ -17,9 +17,11 @@
 version_added: "1.3"
 short_description: create, delete, or modify an Amazon rds instance
 description:
-     - Creates, deletes, or modifies rds instances.  When creating an instance it can be either a new instance or a read-only replica of an existing
-       instance. This module has a dependency on python-boto >= 2.5. The 'promote' command requires boto >= 2.18.0. Certain features such as tags rely
-       on boto.rds2 (boto >= 2.26.0)
+    - Creates, deletes, or modifies rds instances.
+    - When creating an instance it can be either a new instance or a read-only replica of an existing instance.
+    - This module has a dependency on python-boto >= 2.5. The 'promote' command requires boto >= 2.18.0. Certain features such as tags rely
+      on boto.rds2 (boto >= 2.26.0)
+    - Please use boto3 based M(rds_instance) instead.
```

Comment:
The documentation `Creates, deletes, or modifies rds instances...` wasn't really true before as the module also manages snapshots and facts. For thoroughness maybe also add `rds_snapshot`, `rds_snapshot_facts`, `rds_instance_facts` along with `rds_instance` as the modules to use instead (or in the deprecation message if not here).

I'm in favor of just deprecating this module, but let's get rds_snapshot in first.

Reply:
Completely agree with this - suggests the need for a new PR to get the references to the other three modules in.